stdlib/ballerina-http/src/main/ballerina/http/commons.bal

```diff
@@ -129,3 +129,36 @@ function populateRequestFields (Request originalRequest, Request newRequest)  {
     newRequest.userAgent = originalRequest.userAgent;
     newRequest.extraPathInfo = originalRequest.extraPathInfo;
 }
+
+function populateMultipartRequest(Request inRequest) returns Request {
+    string reqContentType;
+    mime:Entity[] reqBodyParts;
+    if (isMultipartRequest(inRequest)) {
+        mime:MediaType mediaType = check mime:getMediaType(inRequest.getContentType());
+        reqContentType = mediaType.primaryType + "/" + mediaType.subType;
```

Comment:
We need to pass the whole value of inRequest.getContentType() instead of just the primary/subtype otherwise boundary parameter will get dropped